projects/ama-sdk/src/lib/api-implementations/acm-api/model-variations/dmn-api-variation.ts

```diff
@@ -0,0 +1,44 @@
+ /*!
+ * @license
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Injectable } from '@angular/core';
+import { ModelApiVariation } from '../model-api';
+ import { DecisionTableContent, Model } from 'ama-sdk/src/lib/api/types';
+
+@Injectable()
+export class DmnApiVariation<M extends Model, C extends DecisionTableContent> implements ModelApiVariation<M, C> {
+
+    readonly contentType: string = 'application/json';
```

Comment:
Although I don't think this file is even needed, but I think the dmn files' content is xml, not json, isn't it?